include/tvm/relax/attrs/ccl.h

```diff
@@ -40,6 +40,14 @@ struct AllReduceAttrs : public tvm::AttrsNode<AllReduceAttrs> {
   }
 };  // struct AllReduceAttrs
 
+/*! \brief Attributes used in allgather operators */
```

Comment:
Since the `AllGatherAttrs` is no longer used, let's remove its definition as well.